testData/ontologies/flat-apptology.json

```diff
@@ -0,0 +1,18 @@
+{
+    "transform": [
+        {
+            "type": "fluid.transforms.value",
+            "inputPath": "http://registry\\.gpii\\.net/common/magnifierEnabled",
+            "outputPath": "http://registry\\.gpii\\.net/common/magnifierApplication"
+        }, {
+            "type": "fluid.transforms.value",
+            "inputPath": "http://registry\\.gpii\\.net/common/magnification",
+            "outputPath": "http://registry\\.gpii\\.net/common/magnifierApplication"
+        }
+    ],
+    "http://registry\\.gpii\\.net/common/screenReaderApplication": "http://registry\\.gpii\\.net/common/screenReaderTTSEnabled",
+    "http://registry\\.gpii\\.net/common/stickyKeysApplication": "http://registry\\.gpii\\.net/common/stickyKeys",
+    "http://registry\\.gpii\\.net/common/slowKeysApplication": "http://registry\\.gpii\\.net/common/slowKeysEnable",
+    "http://registry\\.gpii\\.net/common/debounceKeysApplication": "http://registry\\.gpii\\.net/common/debounceEnable",
```

Comment:
I'm starting to worry about the actual process of deduplicating by "apptology". How does it deal with the case where, say, an application matches multiple entries in the "apptology" and these have a proper overlap with those in another application's? It seems like the canopy algorithm itself should be used here too.
E.g. app 1 does both "stickyKeys" and "slowKeys" and app 2 does both "slowKeys" and "debounceKeys" - what do/should we do then?